src/app/containers/LoginPage/index.js

```diff
@@ -60,14 +65,28 @@ function LoginPage({ history }) {
   );
 
   const handleClosePopup = React.useCallback(() => {
-    setPopupState(false);
-  }, []);
+    if (popupState.isOpen && popupState.isClose) {
+      setPopupState({ isOpen: true, isClose: false });
+    } else if (popupState.isOpen) {
+      setPopupState({ isOpen: false, isClose: true });
+    } else {
+      setPopupState({ isOpen: false, isClose: false });
+    }
+    setPopupState({ isOpen: false, isClose: true });
+    return popupState;
+  }, [popupState]);
+
+  const popupController = {
+    status: popupState,
+    closePopup: handleClosePopup,
+  };
 
   return (
     <MainPanel.Container>
-      {popupState && (
+      {apiProgress === API_PROGRESS.REQ ? <Loading /> : null}
+      {(popupState.isOpen || popupState.isClose) && (
         <Popup
-          onClose={handleClosePopup}
+          onClose={popupController}
```

Comment:
in stead of using logic `popupState && <Popup />`, can you try `<Popup show={logic} />` then you can control the animation inside the Popup